_posts/2019-04-01-vuzix-blade.md

```diff
@@ -103,23 +113,27 @@ It's like someone is following you from behind with a projector and is projectin
 Hold a smartwatch right in front of you in a readable position.
 It's kinda like that, but transparent and without losing the functionality of one of your arms.
 
-So instead of describing it to people, we just put it on their face and they just are immediately captivated by what they're experiencing.
-It takes a moment to learn how to switch your eyes's focus on the heads-up display and back to your surroundings.
-Once you master, this it becomes very natural to interact with the display.
-Seeing that transparent interface is something a lot of people really have to take time to wrap their heads around.
+
+So instead of describing it to people, we just put it on their face and they are just immediately captivated by what they're experiencing.
+It takes a moment to learn how to switch your eyes's focus on the heads up display and back to your surroundings.
+Once you master this it becomes very natural to interact with the display, however staring at it for prolonged periods is not what's it's meant for.
+Seeing that transparent interface for the first time is something a lot of people really have to take time to wrap their heads around the idea.
 After this we show them some pretty pictures with a variety of colors.
-Really brings everything to life and shows off the unexpectedly good visual qualities of the display.
+Really shows off the unexpectedly good visual qualities of the display and brings everything to life!
 
 For a concept device it really shows what the technology is capable of.
-It's capable of running basic Android apps.
-The same ones you would expect to run on a smartwatch.
-Enough computing power to handle the video calling.
-Running TensorFlow Lite to do object detection and classification proved to be a bit too challenging.
+We do hope that the final hardware specs will be a bit more beefy. 
+Imagine running TensorFlow lite object detection in full force on the device. So many cool things we could do with computer vision!
+
+Adding a sim card slot and a GPS chip would also be awesome. 
```

Comment:
... awesome, since this would allow us to autonomously use the glasses without a companion smartphone.